diff --git "a/src/gui/rhi/qrhid3d11.cpp" "b/src/gui/rhi/qrhid3d11.cpp"
index 3faae39a42..534a018321 100644
--- "a/src/gui/rhi/qrhid3d11.cpp"
+++ "b/src/gui/rhi/qrhid3d11.cpp"
@@ -1,6 +1,7 @@
 // Copyright (C) 2019 The Qt Company Ltd.
 // SPDX-License-Identifier: LicenseRef-Qt-Commercial OR LGPL-3.0-only OR GPL-2.0-only OR GPL-3.0-only
 
+#include <initguid.h>
 #include "qrhid3d11_p.h"
 #include "qshader.h"
 #include "vs_test_p.h"
