--- a/modules/core/src/utils/filesystem.cpp	2022-12-28 22:31:52.000000000 +0800
+++ b/modules/core/src/utils/filesystem.cpp	2023-06-01 21:49:07.229751400 +0800
@@ -439,7 +439,7 @@
         cv::String default_cache_path;
 #ifdef _WIN32
         char tmp_path_buf[MAX_PATH+1] = {0};
-        DWORD res = GetTempPath(MAX_PATH, tmp_path_buf);
+        DWORD res = GetTempPathA(MAX_PATH, tmp_path_buf);
         if (res > 0 && res <= MAX_PATH)
         {
             default_cache_path = tmp_path_buf;
