diff --git "a/D:\\vcpkg\\buildtrees\\vtk\\src\\VTK-9-c2b28034b0.clean\\ThirdParty\\ioss\\vtkioss\\Ioss_GroupingEntity.C.orig" "b/D:\\vcpkg\\buildtrees\\vtk\\src\\VTK-9-c2b28034b0.clean\\ThirdParty\\ioss\\vtkioss\\Ioss_GroupingEntity.C"
index 89158bec9..0be7cbad5 100644
--- "a/D:\\vcpkg\\buildtrees\\vtk\\src\\VTK-9-c2b28034b0.clean\\ThirdParty\\ioss\\vtkioss\\Ioss_GroupingEntity.C.orig"
+++ "b/D:\\vcpkg\\buildtrees\\vtk\\src\\VTK-9-c2b28034b0.clean\\ThirdParty\\ioss\\vtkioss\\Ioss_GroupingEntity.C"
@@ -356,7 +356,7 @@ bool Ioss::GroupingEntity::equal_(const Ioss::GroupingEntity &rhs, const bool qu
   if (this->entityState != rhs.entityState) {
     if (!quiet) {
       fmt::print(Ioss::OUTPUT(), "GroupingEntity: entityState mismatch ([] vs. [])\n",
-                 this->entityState, rhs.entityState);
+                 int(this->entityState), int(rhs.entityState));
     }
     return false;
   }
